components/shared/identity/IdentityIndex.vue

```diff
@@ -1,6 +1,5 @@
 <template>
-  <component
-    :is="is"
+  <div
```

Comment:
This won't work with inline

Reply:
It seems like it doesn't matter if the component uses `div` or `span` because the component has its style, `is-flex-wrap-wrap is-flex-grow-1`. For example in carousel-item and gallery-item. So I decided to remove them

Or did I miss somewhere?

Reply:
> Or did I miss somewhere?

I would check where `inline` prop is used :)

Reply:
I've prepared another PR to remove unused props https://github.com/preschian/nft-gallery/pull/64/files
so far, it seems like unused props are safe to remove (inline, no-overflow, vertical-align)